envs/gubbins.yml

```diff
@@ -0,0 +1,71 @@
+name: gubbins
```

Comment:
This should be minimal, not the full conda list. See the other environments for an example